cypress/integration/models/keycloak/users/user.ts

```diff
@@ -166,14 +167,15 @@ export class User {
     }
 
     validateAnalyzeButton(isAccessible: boolean) {
```

Comment:
@ibragins I think the class user is a model to create users using the key cloak UI .
It should just have methods to create /edit/delete users from keycloack console .

The methods to validate presence of buttons should come from their own models .
Just like you added validateAvailableActions() in analysis .
For example : validateAnalyzebutton() should be part of analysis model and test will call that method .

Reply:
Got you, I will fix that